source/utilities.tex

```diff
@@ -17415,12 +17415,14 @@
 
 \pnum
 \remarks
-Call expressions where
-\tcode{id >= num_args_} or
+A call to this function is a core constant expression only if
+\begin{itemize}
+\item
+\tcode{id < num_args_} is \tcode{true} and
+\item
 the type of the corresponding format argument
-(after conversion to \tcode{basic_format_arg<Context>})
-is not one of the types in \tcode{Ts...}
-are not core constant expressions\iref{expr.const}.
+(after conversion to \tcode{basic_format_args<Context>}) is one of the types in \tcode{Ts...}.
```

Comment:
```suggestion
(after conversion to \tcode{basic_format_arg<Context>}) is one of the types in \tcode{Ts...}.
```

We convert each argument to `basic_format_arg<Context>`, not to the container type `basic_format_args<Context>`.